### ISSUE TITLE
Aurelia starts twice when it is embedded

### ISSUE THREAD

**Issue by Tarpsvo** [aurelia/bootstrapper-webpack#13]
**I'm submitting a bug report**
- **Library Version:**
  1.0.0

**Please tell us about your environment:**
- **Operating System:**
  Windows 10
- **Node Version:**
  4.6.0
- **NPM Version:**
  3.8.9
- **JSPM OR Webpack AND Version**
  webpack 2.1.0-beta.22
- **Browser:**
  all
- **Language:**
  all

**Current behavior:**

When Aurelia is embedded in another site, Aurelia tries to initialize itself twice, and succeeds.

**Expected/desired behavior:**

Aurelia should only initialize itself once per page.
- **What is the motivation / use case for changing the behavior?**

Allow embedding of Aurelia apps.

Works on regular Aurelia bootstrapper, doesn't on webpack bootstrapper.

**Comment by stevies** [comment]
Sample code for this - attempting to embed aurelia app inside a Spring Boot / MVC / Thymeleaf template:

```
<!DOCTYPE html>
<html xmlns:th="http://www.thymeleaf.org" th:include="layouts/base :: page" th:lang="en" lang="en">
<section th:fragment="content" id="content" aurelia-app="main">
    <script type="text/javascript" src="aurelia-bootstrap.bundle.js"></script>
    <script type="text/javascript" src="aurelia.bundle.js"></script>
    <script type="text/javascript" src="vendor.bundle.js"></script>
    <script type="text/javascript" src="app.bundle.js"></script>
</section>
</html>
```

Aurelia loads twice.  
I've worked around it with a simple guard in aurelia's main.js - eg:

```
export async function configure(aurelia) {
  if (window.AURELIA_IS_RUNNING) {
    return;
  }

  window.AURELIA_IS_RUNNING = true;
  ...

```

**Comment by Tarpsvo** [comment]
I suspect this is the cause: https://github.com/aurelia/bootstrapper-webpack/commit/54d385c58fcd570f197b65ddc84d50bdb4dbca9b

**Comment by niieani** [comment]
No, that is not the cause. The cited commit is only PAL initialization, and can only happen once (even if invoked multiple times).
I believe the cause is that you have used `aurelia-app="main"`, which means the app will start every time the bootstrapper is run. For your use case, you should can manually start the app instead of relying on the automatic start via the `aurelia-app` parameter.

**Comment by stevies** [comment]
How do you manually start the app?  

In my case, I don't know why the bootstrapper is running more than once.

**Comment by niieani** [comment]
This is how you manually bootstrap (without `aurelia-app` in the body): [example](https://github.com/aurelia/skeleton-navigation/blob/df557fa53bd8412aa58a0466d5b56da53e1dd4a8/skeleton-typescript-webpack/src/main.ts).
We used to manually bootstrap in the Webpack skeleton, but we switched to the `aurelia-app` way to make it consistent with other skeletons.

Does this solve the issue and we can close it?
